Mathlib/Tactic/FindSyntax.lean

```diff
@@ -74,20 +74,23 @@ elab "#find_syntax " id:str d:(&" approx")? : command => do
       let ls := extractSymbols cinfo.value!
       if !declName.isInternal && !ls.isEmpty then symbs := symbs.insert (declName, ls)
   -- From among the parsers in `symbs`, we extract the ones whose `symbols` contain the input `str`
-  let mut msgs := #[]
+  let mut match_results : NameMap (Array (Name × String)) := {}
   for (nm, ar) in symbs.toList do
     let rem : String := " _ ".intercalate (ar.map litToString).toList
     -- If either the name of the parser or the regenerated syntax stub contains the input string,
     -- then we include an entry into the final message.
     if 2 ≤ (nm.toString.splitOn id.getString).length || 2 ≤ (rem.splitOn id.getString).length then
-      msgs := msgs.push <| .ofConstName nm ++ m!":\n  '{rem.trim}'\n"
+      let mod := (← findModuleOf? nm).getD (← getMainModule)
+      match_results := match_results.insert mod <| (match_results.findD mod #[]).push (nm, rem.trim)
   -- We sort the messages to produce a more stable output.
```

Comment:
I guess we dropped this sorting, because `NameMap` does it for us?

Reply:
Good catch!  It was dropped by mistake and I just re-added it!